.github/workflows/build_wheel.py

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env python3
+
+import os
+import sys
+from pathlib import Path
+import subprocess
+
+ROOT = Path(__file__).parent.parent.parent
+
+# For macOS and Windows and linux, run Maturin against the Python interpreter that's
+# been installed and configured for this CI run, i.e. the one that's running
+# this script.
+# Note the docker image konstin2/maturin:master does not work.
+
+os.chdir("./")
+command = [
+    "maturin",
+    "build",
+    "--release",
+    "--interpreter",
+    sys.executable,
+]
+if len(sys.argv) > 1 :
```

Comment:
It looks like the argument is always passed. Can we removed this ?

Reply:
Yes, i have removed the If condition.